clients/src/main/java/org/apache/kafka/clients/consumer/MockConsumer.java

```diff
@@ -153,8 +153,9 @@ public void unsubscribe() {
         for (Map.Entry<TopicPartition, List<ConsumerRecord<K, V>>> entry : this.records.entrySet()) {
             if (!subscriptions.isPaused(entry.getKey())) {
                 List<ConsumerRecord<K, V>> recs = entry.getValue();
-                if (!recs.isEmpty())
+                if (!recs.isEmpty()) {
```

Comment:
You're right, we generally skip them. Confluent's code follows different conventions, so I get them mixed up sometimes, thanks for catching that!